.github/workflows/ci.yml

```diff
@@ -473,3 +480,65 @@ jobs:
         with:
           name: droid-${{ matrix.api-level }}-testapp${{ matrix.unity-version }}-screenshot
           path: samples/artifacts/builds/Android/screen.png
+
+  ios-smoke-test-build:
+    needs: [build]
+    name: Build Smoke Test Ipa - Unity ${{ matrix.unity-version }}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+         unity-version: [2019.4.33f1, 2020.3.21f1, 2021.1.26f1]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+
+      - name: Download XCode app artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: xcode-artifact-${{ matrix.unity-version }}
+          path: samples/artifacts/builds/iOS/Xcode
+
+      - name: Build iOS package
+        shell: pwsh
+        run: ./Scripts/smoke-test-ios.ps1 Build
+
+      - name: Preparing iOS test app for smoke test.
+        uses: actions/upload-artifact@v2
+        with:
+          name: xcode-${{ matrix.unity-version }}-ipa
+          path: |
+            samples/artifacts/builds/iOS/Xcode/archieve/Unity-iPhone/Build/Products/Release-iphonesimulator/
+            !**/Release-iphonesimulator/*.dSYM
+            !**/Release-iphonesimulator/UnityFramework.framework/*
+
+  ios-smoke-test-run:
+    needs: [ios-smoke-test-build]
+    name: Run Smoke Test - iOS Group ${{ matrix.group-id }} Unity ${{ matrix.unity-version }}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+         unity-version: [2019.4.33f1, 2020.3.21f1, 2021.1.26f1]
+         group-id: [1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
sorry but what is the group for?

Reply:
There are 26 devices available for testing and what I did was run 3 devices on each job, where the groups are named by the group Id.

Maybe I should test less devices or use another name approach , ie: "iPhone 8 to iPhone 11" , "iPad Pro (9.7-inch) to iPad Pro (11-inch) (2nd generation)",...

Reply:
Can this be surfaced as the iOS versions instead for clarity?

Reply:
Can we run tests targeting versions instead of devices?